incubator/fluentd-elasticsearch/Chart.yaml

```diff
@@ -1,5 +1,5 @@
 name: fluentd-elasticsearch
-version: 0.4.2
+version: 0.6.0
```

Comment:
Why the jump from 0.4.2 to 0.6.0?

Reply:
this branch is not rebased with master to make the life of reviewer easier. The [current version](https://github.com/kubernetes/charts/blob/master/incubator/fluentd-elasticsearch/Chart.yaml) is 5.0

Reply:
Ah. Makes sense.